### ISSUE TITLE
Extract footnote?

### ISSUE THREAD

**Issue by quad9** [ankushshah89/python-docx2txt#27]
Hi.
How extract Footnote?
Now developing?

**Comment by ShayHill** [comment]
I have a fork that will extract footnotes: docx2python.

**Comment by quad9** [comment]
I want to extract Footnote, but here site, https://docx2python.readthedocs.io/en/latest/index.html#return-value, have no sample.

```
docx2python('path/to/file.docx')
```

but, Here is MSWord's instance.
Following, how do I output Footnote?

**Comment by ShayHill** [comment]
from docx2python import docx2python
from docx2python.iterators import iter_paragraphs

content = docx2python('path/to/file.docx')
footnotes = '\n\n'.join(iter_paragraphs(content.footnotes))


-------- Original message --------

**Comment by quad9** [comment]
I try your sample code.

```
import glob
from docx2python import docx2python
from docx2python.iterators import iter_paragraphs

file_path = glob.glob("./*.docx")
file = file_path[0]
content = docx2python(file)
footnote = '\n\n'.join(iter_paragraphs(content.footnotes))
print(footnote)
```

result is here.

```
$ python test.py
        Traceback (most recent call last):
          File "test.py", line 21, in <module>
            content = docx2python(file)
        TypeError: 'module' object is not callable
```

I try next code.
My modify is `content = docx2python.docx2python(file)`.

```
import glob
from docx2python import docx2python
from docx2python.iterators import iter_paragraphs

file_path = glob.glob("./*.docx")
file = file_path[0]
content = docx2python.docx2python(file)
footnote = '\n\n'.join(iter_paragraphs(content.footnotes))
print(footnote)
```

and result is 'AttributeError'.

```
$ python test.py
    Traceback (most recent call last):
      File "test.py", line 25, in <module>
        fn = '\n\n'.join(iter_paragraphs(content.footnotes))
    AttributeError: 'DocxContent' object has no attribute 'footnotes'
```

Sorry, I can not solve this problem.

**Comment by ShayHill** [comment]
I apologize. Footnote functionality was brand new. I made an error in my pypi upload. The code will work as posted now if you update to docx2python 1.17. Thank you for the catch!!

**Comment by quad9** [comment]
Hi.
I have updated 'docx2python 1.18'. 
I have success getting footnotes.
Thank you!!

**Comment by ShayHill** [comment]
Thank YOU, Nobuyuki! I regret the frustration I caused you with my upload error. Thanks to you, I was able to identify it early.

If you like the project, it would mean very much to me if you would “star” my github project page.

https://github.com/ShayHill/docx2python

I am hoping to further my career as a developer, and your star would translate to food on my family’s table.

Thank you,

  -Shay Hill
